migrations/sqls/20220120941132-update-charge-elements-up.sql

```diff
@@ -0,0 +1,45 @@
+create type water.charge_element_water_model as enum ('no model', 'tier 1', 'tier 2');
```

Comment:
I still don't get why we didn't just do [0,1,2] here but fine whatever - not a big deal

Reply:
just want avoid any mapping